lib/commands/test.js

```diff
@@ -1,24 +1,41 @@
 'use strict';
 
-let TestCommand = require('ember-cli/lib/commands/test');
-let Builder = require('../models/builder');
-let BuildForTestTask = require('../tasks/build-for-test');
-let TestTask = require('../tasks/test');
-let TestServerTask = require('../tasks/test-server');
+const TestCommand = require('ember-cli/lib/commands/test');
+const { emberTestBuildPath } = require('../utils/build-paths');
 
 module.exports = TestCommand.extend({
   name: 'electron:test',
   description: 'Runs your app\'s test suite in Electron.',
 
-  init() {
-    this._super(...arguments);
-    // Used if we are running in server mode
-    this.Builder = Builder;
-    this.tasks.TestServer = TestServerTask;
-    // Used if we're not running in server mode
-    this.tasks.Build = BuildForTestTask;
-    this.tasks.Test = TestTask;
-  },
+availableOptions: [
+    {
+      name: 'environment',
+      type: String,
+      default: 'test',
+      aliases: ['e'],
+      description: 'Possible values are "development", "production", and "test".',
+    },
+    {
+      name: 'config-file',
+      type: String,
+      default: 'testem-electron.js',
+      aliases: ['c', 'cf']
+    },
+    { name: 'server', type: Boolean, default: false, aliases: ['s'] },
+    { name: 'filter', type: String, aliases: ['f'], description: 'A string to filter tests to run' },
+    { name: 'module', type: String, aliases: ['m'], description: 'The name of a test module to run' },
+    { name: 'watcher', type: String, default: 'events', aliases: ['w'] },
+    {
+      name: 'reporter',
+      type: String,
+      aliases: ['r'],
+      description: 'Test reporter to use [tap|dot|xunit] (default: tap)',
+    },
+    { name: 'silent', type: Boolean, default: false, description: 'Suppress any output except for the test report' },
+    { name: 'testem-debug', type: String, description: 'File to write a debug log from testem' },
+    { name: 'test-page', type: String, description: 'Test page to invoke' },
+    { name: 'query', type: String, description: 'A query string to append to the test page URL.' },
+  ],
```

Comment:
This appears to be mostly (all?) duplicated from [`ember-cli/lib/commands/test.js`](https://github.com/ember-cli/ember-cli/blob/b0e01db47ae7e369d4acb937d0d580dae80e9d84/lib/commands/test.js#L22-L79). Is there an easy way we could import it from there instead?

Reply:
Good idea, I just pushed a commit to do that for this as well as the build command